pkg/injector/profiling.go

```diff
@@ -41,3 +41,13 @@ func webhookTimeTrack(start time.Time, timeout time.Duration, success *bool) {
 	metricsstore.DefaultMetricsStore.InjectorRqTime.
 		WithLabelValues(fmt.Sprintf("%t", *success)).Observe(elapsed.Seconds())
 }
+
+// Time tracking function for xds certificate issuance.
+// Will report the time taken to issue an xds certificate
+func certXdsIssueTimeTrack(elapsed time.Duration, commonName string, success *bool) {
```

Comment:
We don't need this anymore right?

Reply:
I had a typ in my commit and then it didn't get pushed. Take care of